report_alternative_layout/report/report_template.xml

```diff
@@ -0,0 +1,155 @@
+        <xpath
+            expr="//div[hasclass('o_company_tagline') and hasclass('mw-50') and hasclass('fw-bold')]"
+            position="attributes"
+        >
+            <attribute name="style">display: none;</attribute>
+        </xpath>
+        <xpath
+            expr="//div[hasclass('o_company_tagline') and hasclass('mw-50') and hasclass('fw-bold')]"
+            position="after"
+        >
+            <!-- Configure Document Layout would throw an error without this condition -->
+            <t t-if="report">
+                <t t-set="report_partner" t-value="report._get_report_partner(o)" />
+                <t
+                    t-set="report"
+                    t-value="report.with_context(lang=report_partner.lang)"
+                />
+                <div class="o_company_tagline mw-50 fw-bold">
+                    <!-- Display the header only if 'report_header' is set.
+                    Handle auto-inserted <p><br></p> from HTML fields -->
+                    <t
+                        t-if="company.report_header and company.report_header != '&lt;p&gt;&lt;br&gt;&lt;/p&gt;'"
+                    >
+                        <span t-field="company.report_header" name="moto" />
+                    </t>
+                    <!-- Display the document number in header -->
+                    <t t-if="report and report.show_document_number">
+                        <div class="col text-end" t-if="o.name">
+                            <strong>
+                                Document No:
+                                <span t-field="o.name" />
+                            </strong>
+                        </div>
+                    </t>
+                    <!-- Display the date field in header -->
+                    <t t-if="report and report.date_field_id and o">
+                        <t t-set="date_value" t-value="report._get_date_value(o)" />
+                        <t t-if="date_value">
+                            <div class="col text-end">
+                                <span t-esc="report._get_date_field_label()" />:<span
+                            >&#xa0;</span>
+                                <span t-esc="date_value" />
+                            </div>
+                        </t>
+                    </t>
+                </div>
+            </t>
+        </xpath>
+        <!-- Hide company address in the header if alternative layout is selected -->
+        <xpath expr="//div[@name='company_address']/ul" position="attributes">
+            <attribute name="t-if">not apply_alternative_layout</attribute>
+        </xpath>
+        <!-- Display address in the header when 'show_address_in_header' is enabled -->
+        <xpath
+            expr="//div[hasclass('row')][div[@name='company_address']]"
+            position="after"
+        >
+            <div class="row">
+                <div class="col-12">
+                    <t t-if="show_address_in_header">
```

Comment:
Should this be the parent of `<div class="row">` two lines above?

Reply:
Even the display style is the same but using as a parent of <div class="row"> is better in my opinion.
Updated.